src/commands/command.json

```diff
@@ -0,0 +1,27 @@
+{
+    "COMMAND":{
+        "summary": "Get array of Redis command details",
+        "complexity": "O(N) where N is the total number of Redis commands",
+        "since": "2.8.13",
+        "group": "server",
```

Comment:
@guybe7 @itamarhaber i wouldn't sort things in the json file alphabetically at all.
i think the short one liners should be at the top, and the long ones at the bottom.
summary should probably always be the first one, then the other one liners (since, group, function, arity, etc), then the big ones by this order: [history, keyspecs, returns, arguments].
i think this will be most convenient to edit.